freqtrade/rpc/api_server/webserver.py

```diff
@@ -125,6 +151,11 @@ def configure_app(self, app: FastAPI, config):
         app.include_router(api_backtest, prefix="/api/v1",
                            dependencies=[Depends(http_basic_or_jwt_token)],
                            )
+        if self._config.get('api_server', {}).get('enable_message_ws', False):
+            logger.info("Enabling Message WebSocket")
+            app.include_router(ws_router, prefix="/api/v1",
+                               dependencies=[Depends(get_ws_token)]
+                               )
```

Comment:
I don't see the websocket part as optional if i'm honest. 

I'm taking the assumption that dataframe transfer is not the only purpose - but the main purpose will be faster communication, also with other partecipants (like fill notifications).

Even if that's not yet available - it'd mean that this setting will have to be removed once the frontend assumes this is working - so the whole "enable_message_ws" setting seems quite pointless to me, as it'd be a temporary thing until it's a requirement for freqUI - at which point we'd have to remove this setting anyway.